### PR TITLE
fix(ctb): remove DefaultValues

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Unnecessary library, only being used in a single place now and it can
just be an internal constant. I see no strong reason for this library to
continue to exist.

### CODE REVIEW

**Review by maurelian — APPROVED** [review]

File: packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol [hunk 1]
```diff
@@ -153,7 +159,7 @@ abstract contract CrossDomainMessenger is
      */
     function xDomainMessageSender() external view returns (address) {
         require(
-            xDomainMsgSender != DefaultValues.DEFAULT_XDOMAIN_SENDER,
+            xDomainMsgSender != DEFAULT_XDOMAIN_SENDER,
             "xDomainMessageSender is not set"
```
Comment (maurelian) [inline packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol:163]: linter error is here

**Comment by tynes** [comment]
Since `xDomainMessageSender` reverts, this is a good change. It doesn't need to be in a lib for the caller to do a comparison against

**Review by tynes — CHANGES_REQUESTED** [review]
needs a linter fix

**Comment by smartcontracts** [comment]
> needs a linter fix

Resolved

**Review by tynes — APPROVED** [review]

**Pull request merged**